tests/dapr_tests.mk

```diff
@@ -313,6 +313,8 @@ setup-test-components: setup-app-configurations
 	$(KUBECTL) apply -f ./tests/config/app_topic_subscription_routing.yaml --namespace $(DAPR_TEST_NAMESPACE)
 	$(KUBECTL) apply -f ./tests/config/app_topic_subscription_routing_grpc.yaml --namespace $(DAPR_TEST_NAMESPACE)
 	$(KUBECTL) apply -f ./tests/config/app_pubsub_routing.yaml --namespace $(DAPR_TEST_NAMESPACE)
+	$(KUBECTL) apply -f ./tests/config/mosquitto.yaml --namespace $(DAPR_TEST_NAMESPACE)
```

Comment:
Please, keep the test independent of mqtt, see how we have DAPR_TEST_PUBSUB as an env var to configure which pubsub to use for E2E tests. The e2e tests should pass independent of a component. For component specific test, we have certification tests in components-contrib.

Reply:
I think it is not really mqtt specific, as there are other brokers supported by dapr, which also support wildcards in the topics, but redis does not, so I am not sure what would be the best solution here.
If we move the test to components-contrib, this has the disadvantage, that we won't notice it if a PR to the main dapr repo breaks this again.